src/arc.ts

```diff
@@ -18,6 +19,7 @@ let selectedProvider: any;
 let web3Modal: Web3Modal;
 let initializedAccount: Address;
 
+let biconomy: any;
```

Comment:
Can we please define the type of biconomy, instead of `any`?

Reply:
Done